UI/Web/src/app/user-settings/manage-reading-profiles/manage-reading-profiles.component.html

```diff
@@ -164,6 +164,8 @@
                               </ng-template>
                               <ng-template #edit>
                                 <app-setting-colour-picker
+                                  [id]="'background-color'"
+                                  [label]="t('background-color-label')"
```

Comment:
These need to be unique. Use a label that has index variable: t('background-color-label', {slotNum: $index})